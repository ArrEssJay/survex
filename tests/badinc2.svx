; include of a file which includes a non existant file
*include badinc
