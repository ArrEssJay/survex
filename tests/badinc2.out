In file included from ./badinc2.svx:3:
./badinc.svx:3:10: error: Couldn't open file "nonexistent_file"
 *include nonexistent_file
          ^~~~~~~~~~~~~~~~
cavern: error: No survey data
